mlir/include/mlir/TableGen/Pattern.h

```diff
@@ -364,12 +390,20 @@ class SymbolInfoMap {
 
     // What kind of entity this symbol represents:
```

Comment:
Could be done in follow up, but I see no reason to the ordering, so we could just sort.

Reply:
I figured properties generally live next to attributes conceptually and historically, but I don't have a strong opinion here

Reply:
I moved the sentinel note to the implementation of the relevant function - you're right, it's not that important.